<launch>

  <arg name="ppl_map" default="/opt/strands/maps/WW_lab_15_09_25-ppl.yaml"/>

  <include file="$(find fake_camera_effects)/launch/camera_effect.launch"/>
  <include file="$(find people_tracker_filter)/launch/filter.launch">
    <arg name="map" value="$(arg ppl_map)"/>
  </include>

  <node name="qsr_lib" pkg="qsr_lib" type="qsrlib_ros_server.py" output="screen" respawn="true"/>
  <node pkg="hrsi_representation" type="online_qtc_creator.py" name="online_qtc_creator" output="screen" respawn="true">
    <param name="ppl_topic" type="string" value="/people_tracker/positions"/>
    <param name="processing_rate" type="int" value="60"/>
    <param name="max_buffer_size" type="int" value="10"/>
    <param name="decay_time" type="double" value="20."/>
    <param name="quantisation_factor" type="double" value="0.01"/>
    <param name="distance_threshold" type="double" value="6.0"/>
    <param name="smoothing_rate" type="double" value="0.3"/>
    <param name="qtc_type" type="int" value="2"/>
    <param name="no_collapse" type="bool" value="false"/>
    <param name="prune_buffer" type="bool" value="false"/>
    <param name="validate" type="bool" value="true"/>
    <param name="goal_topic" type="string" value="/fake_goal"/>
  </node>
  <node pkg="test_cases" type="scenario_server_robot.py" name="scenario_server" output="screen" respawn="true"/>

</launch>
